package.json

```diff
@@ -37,6 +37,7 @@
     "sequelize": "git://github.com/ForestAdmin/sequelize.git#feature/support-schema",
     "superagent": "5.1.0",
     "tedious": "6.4.0",
+    "verror": "^1.10.0",
```

Comment:
Strict versions are really more "secure".
Using `^` can make the dependencies version different across time which is bad for me.
A developer could have a different experience than another because of this.

Also we cannot be sure that `verror` dependencies are properly applying semantic versioning.
They could introduce breaking changes in "patch" version... (Exactly like I badly did for Lumber v2.5.1)